nucliadb/src/nucliadb/search/search/chat/ask.py

```diff
@@ -463,7 +463,7 @@ async def ask(
         ask_request=ask_request,
         find_results=find_results,
         nuclia_learning_id=nuclia_learning_id,
-        predict_answer_stream=predict_answer_stream,
+        predict_answer_stream=predict_answer_stream,  # type: ignore
```

Comment:
@lferran this was the one I mentioned you. `predict_answer_stream` is an `AsyncIterator[GenerativeChunk]` but it expects an `AsyncGenerator[GenerativeChunk, None]`